exercises/Laisha-Calalpa/answers.js

```diff
@@ -0,0 +1,121 @@
+// Question 1
+function greet(name) {
+  return(`Good Morning ${name}`);
```

Comment:
Yep! No parentheses needed around the `return` statement though. Take them out here and for the rest of these problems as well.
```javascript
return `Good Morning, ${name}`;
```

Reply:
Also, we use `camelCase` for JavaScript variables. Not `snake_case`.